src/config/overrides/Workspaces.js

```diff
@@ -123,88 +123,82 @@ export const WORKSPACES = [
         },
         datasetManager: {
           graphIndicators: [
-            { id: 'entities', name: { en: 'Entities', fr: 'Entités' } },
-            { id: 'relationships', name: { en: 'Relationships', fr: 'Relations' } },
-            { id: 'graphKpi0' },
-            { id: 'graphKpi1' },
-            { id: 'graphKpi2' },
-            { id: 'graphKpi3' },
+            { id: 'bars_count', name: { en: 'Bars', fr: 'Bars' }, queryId: 'bars' },
+            { id: 'customers_count', name: { en: 'Customers', fr: 'Clients' }, queryId: 'customers' },
+            {
+              id: 'satisfaction_links_count',
+              name: { en: 'Customers interactions', fr: 'Interactions clients' },
+              queryId: 'satisfaction_graph',
+            },
+            {
+              id: 'relationships_count',
+              name: { en: 'All relationships', fr: 'Tous les liens' },
+              queryId: 'relationships',
+            },
           ],
           categories: [
             {
-              id: 'transport',
-              name: { en: 'Transport', fr: 'Transport' },
-              type: 'relationship',
+              id: 'bars',
+              name: { en: 'Bars', fr: 'Bars' },
+              type: 'entity',
               description: {
                 en:
-                  'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor  incididunt ut ' +
-                  'labore et dolore magna aliqua.',
+                  'Bars are compound entities in the Brewery model. They are responsible of the stock management and ' +
+                  'the number of waiters.\n An entity of type Bar is the parent enity of Customers inside this bar.',
```

Comment:
typo: en**T**ity :slightly_smiling_face:

Reply:
Fixed, thank you ! :)